### ISSUE TITLE
Margin options work only with string keys

### ISSUE THREAD

**Issue by rsov** [prawnpdf/prawn#1011]
Prawn version: 2.2.0

```ruby
class PdfWorksheet < Prawn::Document
  def initialize(doc)
      super('top_margin' => 85) # works
      super(top_margin: 85) # Does not
  end
end
```

The piece of code responsible is at: lib/prawn/document.rb:724

```ruby
sides.zip(positions).each do |side, pos|
  new_margin = options["#{side}_margin"] || (margin[pos] if pos)
  state.page.margins[side] = new_margin if new_margin
end
```

Is it possible to make it accept symbols and strings as options?

Something like:
`new_margin = options["#{side}_margin"] || options["#{side}_margin".to_sym] || (margin[pos] if pos)`

According to the documentation it should take in a key
http://www.rubydoc.info/github/sandal/prawn/Prawn%2FDocument%3Ainitialize

**Comment by bvogel** [comment]
breaks my specs in prawn-templates as well

**Comment by bvogel** [comment]
this was introduced by https://github.com/prawnpdf/prawn/pull/963 and would be (partly) fixed with https://github.com/prawnpdf/prawn/pull/1003

**Comment by pointlessone** [comment]
Fixed in 2.2.1
